plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcPageSourceFactory.java

```diff
@@ -165,6 +166,16 @@ public OrcPageSourceFactory(
         this.fileSystemFactory = requireNonNull(fileSystemFactory, "fileSystemFactory is null");
     }
 
+    public static Properties stripUnnecessaryProperties(Properties schema)
+    {
+        if (isDeserializerClass(schema, OrcSerde.class) && !isFullAcidTable(Maps.fromProperties(schema))) {
+            Properties stripped = new Properties();
+            stripped.put(SERIALIZATION_LIB, schema.getProperty(SERIALIZATION_LIB));
```

Comment:
Should we check if `schema.getProperty(SERIALIZATION_LIB)` exists before putting? (in all 3 methods)

Reply:
Ok. No, just asking